app.js

```diff
@@ -37,9 +37,12 @@ app.use(rateLimiter({
 }))
```

Comment:
There is an argument to be made to have the `rateLimiter` middleware be passed higher up in the chain so that it can prevent other unnecessary middleware from running when an IP has been rate-limited.